lib/ff_dpdk_if.c

```diff
@@ -319,18 +320,29 @@ init_lcore_conf(void)
             printf("\nSkipping non-configured port %d\n", port_id);
             break;
         }
+        lcore_conf.pcap[port_id] = ff_global_cfg.dpdk.port_cfgs[enabled_ports].pcap;
```

Comment:
I thought for a minute that this is not necessary here, it could be below.

Reply:
yes, if a core doesn't process a port, then we don't need to set pcap, but if you set, it is also harmless.

Reply:
Well, please put it in the back. And i'll test it this afternoon.